person.rb

```diff
@@ -1,8 +1,14 @@
-class Person
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'nameable'
+require_relative 'trimmer_decorator'
```

Comment:
- [ ] According to the project requirement, you are meant to run this test to see your decorator in action.

```sh
person = Person.new(22, 'maximilianus')
person.correct_name
capitalized_person = CapitalizeDecorator.new(person)
puts capitalized_person.correct_name
capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
puts capitalized_trimmed_person.correct_name
```
-  [ ] When I ran these, I got an error in the console, I'll suggest you go through your classes and make sure everything is implemented correctly, and then run the code to make sure the result you get is whats expected. :smile: :small_airplane: :wave: 

- - Screenshot for reference:
![Screenshot from 2023-08-29 10-37-54](https://github.com/rabayarabu/OOP-school-library/assets/107632780/33064399-146f-459f-b49d-d65eba1022d9)